### DISCUSSION TITLE
Show line points only in selected/hovered slices

### DISCUSSION THREAD

**Discussion by matobiely** [plouc/nivo#1602] [Q&A] [answered]
I want to show clean line chart, without points, and only on select/hover on slice, point will show. But only in selected slice, so basically only for one x point. I tried many setups, but not successful. 
Maybe it can by solved by custom layer, but i am not so advanced in nivo to do this and there is missing guide for custom layer so i do not even know, where i should start

**Comment by matobiely** [comment]
I cannot edit issue, so here is image, what exactly i mean

![image](https://user-images.githubusercontent.com/64771567/122415897-92c7c680-cf88-11eb-9157-5a5d5f3ad31a.png)

**Comment by wyze** [comment, marked as answer]
Here is an example using a custom layer to achieve what you are looking for: https://codesandbox.io/s/intelligent-sunset-3rknb?file=/src/App.tsx
